submissions/ustymchyk/html-css-popup/index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML and CSS popup</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav">
+      <a class="header__item" href>Mail</a>
```

Comment:
Wrong class name, it's part of navigation, so it's better name it `nav__item`.

Reply:
Agree, thanks.